sender.go

```diff
@@ -189,15 +196,21 @@ func (s *sender) trySend(ctx context.Context, evt eventer) error {
 				// successful send
 				return err
 			}
-			switch err.(type) {
-			case *amqp.Error, *amqp.DetachError, net.Error:
-				recvr(err)
+			switch e := err.(type) {
+			case *amqp.Error:
+				if e.Condition == errorServerBusy || e.Condition == errorTimeout {
+					// don't rebuild the connection in this case, just delay and try again
+					recvr(err, false)
```

Comment:
Need to return or break after this line?

Reply:
Good catch, added `break`